modules/vmseries/variables.tf

```diff
@@ -140,12 +140,12 @@ variable "custom_image" {
 
 variable "machine_type" {
   description = "Firewall instance machine type, which depends on the license used. See the [Terraform manual](https://www.terraform.io/docs/providers/google/r/compute_instance.html)"
-  default     = "n1-standard-4"
+  default     = "n2-standard-4"
   type        = string
 }
 
 variable "min_cpu_platform" {
-  default = "Intel Broadwell"
```

Comment:
Can we add description for variable e.g. "Specifies a minimum CPU platform for the VM instance."?

Reply:
* Generally, due flex license our solution can support custom machine type. More memory etc, but testing needs to be done or we can introduce it later, up to you.
* I would suggest that add information that number of Cores is maximum number of supported interfaces. It can save time during deployment.